### PR TITLE
Markdown, document the standard markdown flavor

### LINKED ISSUE #156: Add flavor of Markdown (or related guidance thereof)
It is not clear if we are using a particular flavor or Markdown or not. This issue provides guidance about that.

### PR DESCRIPTION
**Pull request by MattDodsonEnglish**
Fixes #156 .

### CODE REVIEW

**Review by JStickler — APPROVED** [review]
LGTM

File: docs/sources/writing-guide/markdown-guide/index.md [hunk 1]
```diff
@@ -17,6 +17,9 @@ keywords:
 
 This Markdown guide helps keep contributions consistent across all Grafana Labs documentation. Refer to the guide and update it as needed when a subject matter expert (SME) answers a question about Markdown syntax, or a decision is made about how to apply Markdown.
 
+Grafana uses the default Hugo Markdown parser, Goldmark.
```
Comment (osg-grafana) [inline docs/sources/writing-guide/markdown-guide/index.md:20]:
```suggestion
Grafana uses the default Hugo Markdown parser named Goldmark.
```

Comment (osg-grafana) [reply docs/sources/writing-guide/markdown-guide/index.md:20]: Alternative recommendation: "called"

Comment (osg-grafana) [reply docs/sources/writing-guide/markdown-guide/index.md:20]: Rationale: use words instead of punctuation to stand for words.

**Review by GrafanaWriter — APPROVED** [review]

**Pull request merged**